src/main/java/com/dev/cinema/dao/impl/UserDaoImpl.java

```diff
@@ -24,9 +28,9 @@ public User add(User user) {
         try {
             session = sessionFactory.openSession();
             transaction = session.beginTransaction();
-            byte[] salt = HashUtil.getSalt();
+            byte[] salt = hashUtil.getSalt();
```

Comment:
hashing should be performed on the service layer

Reply:
Done